app/com/kinja/play/Closure.scala

```diff
@@ -102,29 +126,34 @@ class ClosurePlugin @Inject() (implicit app: Application) extends Plugin {
    * If the app is running in production mode then always returns a same engine instance,
    * otherwise returns a brand new instance.
    */
-  def api: ClosureEngine = {
+  private def api: ClosureEngine = {
     if (engine == null) {
       reloadEngine()
     }
     engine
   }
 
-  override def onStart() = {
```

Comment:
`onStart` was a plugin thing. Modules' start code should simply be in the class body ([see migration guide](https://www.playframework.com/documentation/2.4.x/PluginsToModules)).